fileio/ft_filetype.m

```diff
@@ -1425,10 +1459,6 @@
   type = 'motion_c3d';
   manufacturer = 'https://www.c3d.org';
   content = 'motion capture data';
-elseif filetype_check_extension(filename, '.mvnx') && filetype_check_header(filename, '<?xml')
```

Comment:
It is not clear to me why this piece of code has been deleted

Reply:
Hi @robertoostenveld, Please see the comment below. Thanks!